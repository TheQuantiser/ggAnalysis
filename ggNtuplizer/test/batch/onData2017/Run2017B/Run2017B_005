file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/9CB3CF71-9337-E811-99F1-0CC47A7C3422.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/BAA829E5-9937-E811-9B8E-0CC47A4D7626.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/049ED9EE-9B37-E811-81B3-0025905B85B8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/301DDAF6-B037-E811-BAE6-0025905A60F8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/EEA21C91-9037-E811-95B0-0025905B8564.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/2AB17DC6-B437-E811-9ADA-0025905A48FC.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/228B2370-9337-E811-AF2E-003048FFCBB8.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/268046EE-D537-E811-9E1A-0CC47A4D7698.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/30823970-7D37-E811-A0BE-0025905B8600.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/D6EFD6B4-B437-E811-8ED5-0025905A48C0.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/76819B45-BA37-E811-8114-0025905A6082.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/B018A94C-8437-E811-8DF8-0CC47A4D7692.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/389AEAEB-9937-E811-B42B-0025905B8576.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/843D1CD0-A337-E811-B00A-0025905A60E4.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/8A1EB9CC-F136-E811-8769-0CC47A4C8E1E.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/7C4E60D1-7837-E811-A9F0-0CC47A4D769A.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/02A61DB0-9537-E811-9604-0CC47A4D75F4.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/6ABAD83F-BA37-E811-A468-0025905A6068.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/70FAD4DC-0D37-E811-8737-0CC47A4C8E64.root
file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/BC5FE539-D037-E811-BA38-0CC47A7C3410.root
