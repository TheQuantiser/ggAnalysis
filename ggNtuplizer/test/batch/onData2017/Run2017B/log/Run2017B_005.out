scorpion11.spa.umn.edu
cms.Source("PoolSource",
    fileNames = cms.untracked.vstring('file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/049739E6-9B37-E811-802B-0CC47A4C8ED8.root', 
        'file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/0C8FC10A-EA37-E811-8D45-0025905B860C.root', 
        'file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/6A31B09C-8C37-E811-950A-0CC47A7C340E.root', 
        'file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/DEAFD75E-8D37-E811-8D18-0CC47A4C8F12.root', 
        'file:/local/cms/user/wadud/aNTGCmet/datasets/SinglePhotonRun2017_31Mar2018/Run2017B/SinglePhoton/MINIAOD/31Mar2018-v1/90000/CA0EED5C-C537-E811-A614-0025905A6066.root')
)

******************* RunOnData *******************
************** MC dependence removal ************
removing MC dependencies for photons
removing MC dependencies for electrons
removing MC dependencies for muons
removing MC dependencies for taus
removing MC dependencies for jets
adding L2L3Residual JEC for: patJetCorrFactors
|---- jetToolbox: Initializing collection... (with postfix updated)
|---- jetToolBox: Adding these btag discriminators: ['pfCombinedInclusiveSecondaryVertexV2BJetTags', 'pfCombinedCvsLJetTags', 'pfCombinedCvsBJetTags', 'pfDeepCSVJetTags:probb', 'pfDeepCSVJetTags:probbb', 'pfDeepCSVJetTags:probc', 'pfDeepCSVJetTags:probudsg', 'pfDeepFlavourJetTags:probb', 'pfDeepFlavourJetTags:probbb', 'pfDeepFlavourJetTags:problepb', 'pfDeepFlavourJetTags:probc', 'pfDeepFlavourJetTags:probuds', 'pfDeepFlavourJetTags:probg'] in the jet collection.
|---- jetToolBox: JETTOOLBOX IS UPDATING slimmedJets collection
|---- jetToolBox: Applying these corrections: ('AK4PFchs', ['L1FastJet', 'L2Relative', 'L3Absolute', 'L2L3Residual'], 'None')
|---- jetToolBox: Creating selectedPatJetsAK4PFCHSupdated collection.
************** MC dependence removal ************
removing MC dependencies for jets
Thread 3 (Thread 0x2b018187f700 (LWP 90)):
#0  0x00000030c2e0b68c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
#1  0x00002b013a19b2cc in __gthread_cond_wait (__mutex=<optimized out>, __cond=<optimized out>) at /mnt/build/davidlt/gcc630/b/BUILD/slc6_amd64_gcc630/external/gcc/6.3.0/gcc-tags_gcc_6_3_0_release-243837/obj/x86_64-unknown-linux-gnu/libstdc++-v3/include/x86_64-unknown-linux-gnu/bits/gthr-default.h:864
#2  std::condition_variable::wait (this=<optimized out>, __lock=...) at ../../../../../libstdc++-v3/src/c++11/condition_variable.cc:53
#3  0x00002b01603d278d in Eigen::NonBlockingThreadPoolTempl<tensorflow::thread::EigenEnvironment>::WaitForWork(Eigen::EventCount::Waiter*, tensorflow::thread::EigenEnvironment::Task*) () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/external/slc6_amd64_gcc630/lib/libtensorflow_cc.so
#4  0x00002b01603d31f9 in Eigen::NonBlockingThreadPoolTempl<tensorflow::thread::EigenEnvironment>::WorkerLoop(int) () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/external/slc6_amd64_gcc630/lib/libtensorflow_cc.so
#5  0x00002b01603d1057 in std::_Function_handler<void (), tensorflow::thread::EigenEnvironment::CreateThread(std::function<void ()>)::{lambda()#1}>::_M_invoke(std::_Any_data const&) () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/external/slc6_amd64_gcc630/lib/libtensorflow_cc.so
#6  0x00002b013a1a0c2f in std::execute_native_thread_routine (__p=0x2b017decb2c0) at ../../../../../libstdc++-v3/src/c++11/thread.cc:83
#7  0x00000030c2e07aa1 in start_thread () from /lib64/libpthread.so.0
#8  0x00000030c26e8c4d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2b014af80700 (LWP 83)):
#0  0x00000030c2e0f37d in waitpid () from /lib64/libpthread.so.0
#1  0x00002b013d6fd8e7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/pluginFWCoreServicesPlugins.so
#2  0x00002b013d6fde85 in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/pluginFWCoreServicesPlugins.so
#3  0x00002b013a1a0c2f in std::execute_native_thread_routine (__p=0x2b013cbb6950) at ../../../../../libstdc++-v3/src/c++11/thread.cc:83
#4  0x00000030c2e07aa1 in start_thread () from /lib64/libpthread.so.0
#5  0x00000030c26e8c4d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2b013a4ff6c0 (LWP 64)):
#0  0x00000030c26df403 in poll () from /lib64/libc.so.6
#1  0x00002b013d6fdcd4 in full_read.constprop () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/pluginFWCoreServicesPlugins.so
#2  0x00002b013d6fe4aa in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/pluginFWCoreServicesPlugins.so
#3  0x00002b013d6ff505 in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002b016f1eda13 in EcalClusterToolsT<false>::cluster2ndMoments(std::vector<std::pair<EcalRecHit const*, float>, std::allocator<std::pair<EcalRecHit const*, float> > > const&, double, double, bool) () from /data/cmszfs1/user/wadud/aNTGCmet/CMSSW_9_4_17/lib/slc6_amd64_gcc630/pluginRecoEgammaPhotonIdentificationPlugins.so
#6  0x00002b016f1e6f84 in PhotonIDValueMapProducer::produce(edm::Event&, edm::EventSetup const&) () from /data/cmszfs1/user/wadud/aNTGCmet/CMSSW_9_4_17/lib/slc6_amd64_gcc630/pluginRecoEgammaPhotonIdentificationPlugins.so
#7  0x00002b0138766da3 in edm::stream::EDProducerAdaptorBase::doEvent(edm::EventPrincipal const&, edm::EventSetup const&, edm::ActivityRegistry*, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/libFWCoreFramework.so
#8  0x00002b0138692772 in edm::WorkerT<edm::stream::EDProducerAdaptorBase>::implDo(edm::EventPrincipal const&, edm::EventSetup const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/libFWCoreFramework.so
#9  0x00002b0138642fb7 in decltype ({parm#1}()) edm::convertException::wrap<bool edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::MyPrincipal const&, edm::EventSetup const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}>(bool edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::MyPrincipal const&, edm::EventSetup const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}) () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/libFWCoreFramework.so
#10 0x00002b013864318b in bool edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::MyPrincipal const&, edm::EventSetup const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/libFWCoreFramework.so
#11 0x00002b0138644b2b in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(std::__exception_ptr::exception_ptr const*, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::MyPrincipal const&, edm::EventSetup const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/libFWCoreFramework.so
#12 0x00002b0138645675 in edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute() () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/libFWCoreFramework.so
#13 0x00002b0139ca322c in tbb::internal::custom_scheduler<tbb::internal::IntelSchedulerTraits>::local_wait_for_all (this=0x2b013aa63200, parent=..., child=<optimized out>) at ../../src/tbb/custom_scheduler.h:509
#14 0x00002b01386e3d8c in edm::EventProcessor::readAndProcessEvents() () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/libFWCoreFramework.so
#15 0x00002b01386eea51 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms.cern.ch/slc6_amd64_gcc630/cms/cmssw/CMSSW_9_4_17/lib/slc6_amd64_gcc630/libFWCoreFramework.so
#16 0x000000000040e802 in main::{lambda()#1}::operator()() const ()
#17 0x000000000040d1ba in main ()
